pkg/models/model_helpers.go

```diff
@@ -1,6 +1,9 @@
 package models
 
-import "github.com/lib/pq"
+import (
+	"github.com/google/uuid"
+	"github.com/lib/pq"
+)
 
 // StringPointer returns a pointer to a string input
 func StringPointer(st string) *string {
```

Comment:
Why not simply use &st?

Reply:
Hi @OddTomBrooks , this is also not a change from this ticket, and if we changed it it should be in a different ticket.

Reply:
Fair enough. 👍